feat(changelog): console-added-cli-console 2023-10-30 (#2333)
scaleway/docs-content: changelog/october2023/2023-10-30-console-added-cli-console.mdx

---
title: CLI-Console
status: added
author:
    fullname: 'Join the #cli channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2023-10-30
category: console
product: console
---

> New features: Our CLI works with webassembly and uses your console credentials to offer you a CLI experience without requiring installation on your machine. Use the output formatting and discover how the CLI could be useful for you. You can also install it natively on your local machine, as we support Windows, macOS and Linux.

Reviewer: ```suggestion
New features: Our integrated console CLI works with webassembly and uses your console credentials to offer you a CLI experience without requiring installation on your machine. Use the output formatting and discover how the CLI could be useful for you. You can also install it natively on your local machine, as we support Windows, macOS and Linux.
```

Reviewer: Is it clear enough that we talk about the console CLI ?

> New features: Our integrated console CLI works with WebAssembly and uses your console credentials to offer you a seamless CLI experience without installing anything on your machine. Benefit from enhanced output formatting and discover a new way to manage your Scaleway resources. You can also install it natively on your local machine, as we support Windows, macOS and Linux.

Reviewer 2: "New feature" au singulier non ?

Reviewer 3: ```suggestion
New feature: Our integrated console CLI works with WebAssembly and uses your console credentials to offer you a seamless CLI experience without installing anything on your machine. Benefit from enhanced output formatting and discover a new way to manage your Scaleway resources. You can also install it natively on your local machine, as we support Windows, macOS and Linux.
```

> New features: Our integrated console CLI works with WebAssembly and uses your console credentials to offer you a seamless CLI experience without installing anything on your machine. Benefit from an enhanced output formatting and discover a new way to manage your Scaleway resoures. You can also install it natively on your local machine, as we support Windows, macOS and Linux.

Reviewer 4: ```suggestion
New features: Our integrated console CLI works with WebAssembly and uses your console credentials to offer you a seamless CLI experience without installing anything on your machine. Benefit from enhanced output formatting and discover a new way to manage your Scaleway resources. You can also install it natively on your local machine, as we support Windows, macOS, and Linux.
```

> See our [product documentation](https://www.scaleway.com/en/docs/developer-tools/scaleway-cli/quickstart/) for more information.

Reviewer 5: ```suggestion
See our [product documentation](/developer-tools/scaleway-cli/quickstart/) for more information.
```
